MobileCenterCrashes/MobileCenterCrashesTests/MSCrashesTests.mm

```diff
@@ -579,6 +581,37 @@ - (void)testWarningMessageAboutTooManyErrorAttachments {
   XCTAssertTrue(warningMessageHasBeenPrinted);
 }
 
+- (void)testTrackModelException {
+
+  // If
+  __block NSString *type;
+  __block NSString *errorId;
+  __block MSException *exception;
+  id<MSLogManager> logManagerMock = OCMProtocolMock(@protocol(MSLogManager));
+  OCMStub([logManagerMock processLog:[OCMArg isKindOfClass:[MSAbstractLog class]] forGroupId:OCMOCK_ANY])
+  .andDo(^(NSInvocation *invocation) {
+    MSHandledErrorLog *log;
+    [invocation getArgument:&log atIndex:2];
+    type = log.type;
```

Comment:
You don't need to assign type, errorId and exception to each variable. If you take `log` variable out of the block, you can assign the argument to `log` and use it directly.

Reply:
I kept same style as https://github.com/Microsoft/mobile-center-sdk-ios/blob/develop/MobileCenterAnalytics/MobileCenterAnalyticsTests/MSAnalyticsTests.m#L365 actually.

Reply:
Yep. There are still so many places that need to be reformatted. We are reformatting whenever we touch any of those files. Still on going.